OpenStack-accounting/sql/glance_get_accounting_data.sql

```diff
@@ -0,0 +1,48 @@
+CREATE PROCEDURE `get_accounting_data`(IN starttime datetime, IN endtime datetime)
+BEGIN
+/*
+   This procedure generates accounting data for glance
+*/
+SELECT
+    p.name AS Project,
+    pp.name AS Department,
+    COUNT(g.id) AS Images,
+    ip.value as GlanceType,
+    if(il.value like "%rbd%", "RBD" ,if(il.value like "%swift%","OBJECT","UNKNONWN")) as StorageBackend,
+    @VolumeSeconds:=SUM(IF(g.created_at <= starttime /* Captures Images which were created outside of the period deleted out of the period */
```

Comment:
Should this be `@ImageSeconds` instead of `@VolumeSeconds`?

Reply:
Yes it should. Fixed